cognee/eval_framework/modal_run_eval.py

```
@@ -1,17 +1,20 @@
 import modal
 import os
-import json
 import asyncio
 import datetime
+import hashlib
```

Ruff (0.11.9)

`hashlib` imported but unused

Remove unused import: `hashlib`

(F401)